pkg/dao/interfaces.go

```diff
@@ -13,7 +13,7 @@ type RepositoryDao interface {
 	List(orgID string, paginationData api.PaginationData, filterData api.FilterData) (api.RepositoryCollectionResponse, int64, error)
 	Delete(orgID string, uuid string) error
 	SavePublicRepos(urls []string) error
-	ValidateParameters(orgId string, params api.RepositoryValidationRequest) (api.RepositoryValidationResponse, error)
+	ValidateParameters(orgId string, params api.RepositoryValidationRequest, UUIDList []string) (api.RepositoryValidationResponse, error)
```

Comment:
```suggestion
	ValidateParameters(orgId string, params api.RepositoryValidationRequest, uuids []string) (api.RepositoryValidationResponse, error)
```
imo list is implicit in the type and we can just call this uuids

Reply:
Ahh.. yes. agreed